src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -20,79 +21,99 @@ tags:
 {% endAside %}
 
-A workflow for improving Core Web Vitals for your website will be explored in this guide, but where that workflow begins depends on whether you're collecting your own field data. Where it ends may depend on which of Google's tools you'll find useful in diagnosing and fixing user experience problems.
+For example, Lighthouse is a lab-based tool that runs tests with simulated throttling in a simulated desktop or mobile environment. While such simulations of slower network and device conditions are helpful when trying to diagnose performance problems, [they're just a single slice](/lab-and-field-data-differences/) of the large variety in network conditions and device capabilities and so may not reflect what users on your sites are experiening.
 
-## Collect your own field data if possible
+Lab-based tools like Lighthouse also typically do a "cold load" of a webpage as a totally new visitor. This is often the slowest load, but in real life, visitors may have some assets cached if they have visited before, or when they are browsing around the site. New visitors, and tools, also may experience the site differently with cookie banners or other content present to them.
 
-While lab tools are invaluable, [the data they provide isn't always predictive of how a website performs for real users](https://discuss.httparchive.org/t/lighthouse-scores-as-predictors-of-page-level-crux-data/2232).
+In short, while lab-based tools can give an indication of potential performance problems and help you debug and iterate, they may not represent how many of your visitors actually experience your website. Use field data for measuring real-world performance, and lab-based tools like Lighthouse for diagnostics of how to improve it. See also the [When to use Lighthouse](#when-to-use-lighthouse) section.
 
-For example, Lighthouse runs tests with simulated throttling in a simulated desktop or mobile environment. While such simulations of slower network and device conditions often help draw attention to user experience problems better than native network and device conditions, [they're just a single slice](/lab-and-field-data-differences/) of the large variety in network conditions and device capabilities across a website's entire user base.
+Google measures Core Web Vitals through the [Chrome User Experience Report (CrUX)](https://developer.chrome.com/docs/crux/). This is a public dataset collected from real Chrome users. It is the backbone of many of the Google, and third-party, tools which report a site's Core Web Vitals.
 
-{% Aside %}
-To learn more about how simulated throttling works and how it differs from other types of throttling, [this bit of Lighthouse documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#network-throttling) is your time.
-{% endAside %}
+CrUX has its limitations, though. It can often tell you _when_ there is a problem, but often has insufficient data to understand _why_.
```

Comment:
```suggestion

CrUX has its limitations, though. It can often tell you _when_ there is a problem, but often has insufficient data to tell you _why_.
```